lib/rules/disallow-space-after-object-keys.js

```diff
@@ -134,7 +134,7 @@ module.exports.prototype = {
             var maxKeyEndPos = 0;
             var tokens = [];
             node.properties.forEach(function(property) {
-                if (property.shorthand || property.kind !== 'init') {
+                if (property.shorthand || property.kind !== 'init' || property._babelType === 'SpreadProperty') {
```

Comment:
There was this issue https://github.com/babel/babel-eslint/issues/142

https://github.com/babel/babel-eslint/pull/143/files

But it caused another issue in https://github.com/babel/babel-eslint/issues/142#issuecomment-118707124 so had to keep `node.type === 'Property'` and make a new property to keep info about `SpreadProperty`.

But maybe it's only tied to eslint and not jscs?